frontend/src/DeviceDetailsPage/DeviceDetailsTabTable.jsx

```diff
@@ -41,17 +41,28 @@ export default class DeviceDetailsTabTable extends React.Component {
     const { value } = this.state;
     const { device, tabs } = this.props;
     return (
-      <Container component={Paper}>
+      <Container>
```

Comment:
Why did you remove Paper?

Reply:
I removed it during manual testing to see if it was aesthetically nicer without, but then I re-added them to the individual TabPanel components instead by accident.

Reply:
Actually I tested it out and adding the Paper component to each TabPanel is the right approach.

**If unchanged from before the PR:**
![image](https://user-images.githubusercontent.com/33813045/105919963-860bfb80-6004-11eb-92b1-b31145a05878.png)

**If removed from DeviceDetailsTabTable but added individually to each TabPanel:**
![image](https://user-images.githubusercontent.com/33813045/105920079-b784c700-6004-11eb-8b0b-3e84c5e869d3.png)